### DISCUSSION TITLE
Ability to hide history from library tab?

### DISCUSSION THREAD

**Discussion by masonhfisher** [ReVanced/revanced-discussions#1026] [Question] [answered]
Is it possible for a patch to be added that hides the history (visually) from the library tab?

**Comment by oSumAtrIX** [comment]
You can see a list of patches to apply in ReVanced Manager

**Reply by masonhfisher** [reply]
Sorry if I sound like I'm being difficult but I can't find a patch to hide the history from the library tab

**Reply by LisoUseInAIKyrios** [reply, marked as answer]
Because there is no patch.

It might be possible to hide the history using custom filter, but trying that requires a computer and knowledge of how to use the command line (outside the scope of discussion here).

**Reply by masonhfisher** [reply]
Yeah thought so. Thank you though!
